storage/src/vespa/storage/storageserver/storagenode.h

```diff
@@ -137,25 +133,30 @@ class StorageNode : private config::IFetcherCallback<vespa::config::content::cor
     virtual void configure(std::unique_ptr<document::config::DocumenttypesConfig> config,
                            bool hasChanged, int64_t generation);
     void configure(std::unique_ptr<BucketspacesConfig>) override;
+    void configure(std::unique_ptr<CommunicationManagerConfig> config) override;
 
 protected:
     // Lock taken while doing configuration of the server.
     std::mutex _configLock;
     std::mutex _initial_config_mutex;
     using InitialGuard = std::lock_guard<std::mutex>;
+
     // Current running config. Kept, such that we can see what has been
     // changed in live config updates.
     std::unique_ptr<StorServerConfig> _serverConfig;
     std::unique_ptr<UpgradingConfig> _clusterConfig;
     std::unique_ptr<StorDistributionConfig> _distributionConfig;
     std::unique_ptr<document::config::DocumenttypesConfig> _doctypesConfig;
     std::unique_ptr<BucketspacesConfig> _bucketSpacesConfig;
+    std::unique_ptr<CommunicationManagerConfig> _comm_mgr_config;
+
```

Comment:
Seems to be a pattern here, Perhaps a small utility class with _current and _new config at some point in time.

Reply:
Agreed, many more configs to go... 🥹